.travis.yml

```diff
@@ -14,16 +18,9 @@ addons:
       - libboost-date-time-dev 
       - libboost-test-dev
       - google-mock
- 
-before_script:
-  - "cd /usr/src/gmock && sudo cmake . && sudo cmake --build . && sudo mv libg* /usr/local/lib/ && sudo mv gtest/libg* /usr/local/lib/ ; cd -"
 
 script: ./travis.sh 
 
-branches:
-  only:
-    - master
-
```

Comment:
Not sure what this adds but I guess it's fine.

I'll try and find a way of sending emails to the group by default only for what builds from cucumber/cucumber-cpp repo. Since it wasn't a PR, your appveyor branch in konserw/cucumber-cpp shouldn't have sent notifications there when failing.

I'll merge it ASAP.